src/config.rs

```diff
@@ -44,20 +42,20 @@ fn pip() -> String {
 }
 
 impl Config {
-    /// Load the config from the associated file.
+    /// Load the config file from a users pacdef config folder.
     ///
     /// # Errors
-    ///
-    /// This function will return an error if the config file exists but cannot be
-    /// read, its contents are not UTF-8, or the file is malformed.
-    pub fn load(config_file: &Path) -> Result<Self> {
-        let from_file = read_to_string(config_file);
+    /// - If a config file cannot be found.
+    pub fn load() -> Result<Self> {
+        let config_file = dirs::config_dir()
+            .context("getting the config directory")?
+            .join("pacdef/config.toml");
```

Comment:
I believe `extend` with an array would be more idiomatic here, though I guess I'm just bike-shedding.

Reply:
Hmm, then we'd need `mut` on the declaration since `extend()` takes a `&mut self` argument.

Reply:
Ah, this is `join`, not `push`. Let's leave it like this ig. Looks more concise.